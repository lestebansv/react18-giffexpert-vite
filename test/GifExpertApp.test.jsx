import { render } from '@testing-library/react';
import { GifExpertApp } from '../src/GifExpertApp';

describe('pruebas en <GifExpertApp />', () => {

    test('debe de mostrarse correctamente', () => {
        const { container } = render(<GifExpertApp />);
        expect(container).toMatchSnapshot();
    });

})