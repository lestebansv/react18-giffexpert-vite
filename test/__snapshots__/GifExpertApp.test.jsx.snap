// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <GifExpertApp /> debe de mostrarse correctamente 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      id="inputCategory"
      placeholder="Buscar Gifs"
      type="text"
      value=""
    />
  </form>
  <h3>
    GOT
  </h3>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
